jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -939,14 +940,31 @@ impl TallyPublicStatus {
     }
 }
 
+#[derive(Clone)]
+pub struct TallyPrivateStatus {
+    tally: Vec<u8>,
+}
+
+#[juniper::object(Context = Context)]
+impl TallyPrivateStatus {
+    fn tally(&self) -> String {
+        hex::encode(&self.tally)
+    }
+}
+
 #[derive(Clone)]
 pub enum TallyStatus {
     Public(TallyPublicStatus),
+    Private(TallyPrivateStatus),
 }
 
 graphql_union!(TallyStatus: Context |&self| {
     instance_resolvers: |_| {
-        &TallyPublicStatus => match *self { TallyStatus::Public(ref c) => Some(c) },
+        &TallyPublicStatus => match *self {
+            TallyStatus::Public(ref c) => Some(c),
+            // TODO: resolve this union for private
+            TallyStatus::Private(_) => unimplemented!("..."),
+        },
```

Comment:
@enzoc4 I think you implemented this, or at least have more experience with juniper. 
I do not know what this piece of code is doing nor what I'm supposed to do with the new private stuff here.

Reply:
It was actually @eugene-babichenko who did this part, but I'll check it later.

Here you only need to return `None,` though. The left side (`TallyPublicStatus`) is what you write in the graphql query, and you need to tell it the current variant is of that type.

The counting part is the one that I'm not sure if we want to do something about.

Although, I think we are not using the explorer, right @rinor ?